lib/sift/list.rb

```diff
@@ -0,0 +1,5 @@
+
+module Sift
```

Comment:
I'll remove it. I thought, at first, I could generalize list results. We can deal with this later.